spec/statement.md

```diff
@@ -0,0 +1,176 @@
+# Statement
+
+## Table of Contents
+
+* [Definitions](#definitions)
+  * [Current indentation level](#current-indentation-level)
+  * [Document indentation character](#document-indentation-character)
+* [Syntax](#syntax)
+  * [ScssStatements](#scssstatements)
+  * [IndentedStatements](#indentedstatements)
+  * [Stylesheet](#stylesheet)
+  * [Block](#block)
+  * [Comments](#comments)
+    * [LoudComment](#loudcomment)
+    * [SilentComment](#silentcomment)
+    * [WhitespaceComment](#whitespacecomment)
+  * [Whitespace](#whitespace)
+  * [Indentation](#indentation)
+
+## Definitions
+
+### Current indentation level
+
+The `current indentation level` is the count of the [`documentation indentation
+character`] between a new line and any non-whitespace characters of the last
+consumed statement. Lines that only contain whitespace or do not start a
+statement do not impact the `current indentation level`.
+
+A document parsed in the indented syntax may not begin with whitespace, and the
+initial `current indentation level` for any document is `0`.
+
+> Changes in the indentation level are used by the indented syntax to start and
+> end blocks of statements.
+
+[`documentation indentation character`]: #document-indentation-character
+
+### Document indentation character
+
+The `document indentation character` is defined as the first tab or space
+character used in an [`IndentMore`] production in a document.
+
+[`IndentMore`]: #indentation
+
+> The `document indentation character` is the character used for calculating the
+> [`current indentation level`]. In the indented syntax, no character other than
+> the `document indentation character` may be used for indentation.
+
+[`current indentation level`]: #current-indentation-level
+
+## Syntax
+
+### ScssStatements
+
+<x><pre>
+**ScssStatements**      ::= (Statement ';'?¹)* Statement?
+</pre></x>
+
+1. This production is mandatory unless the previous `Statement` is a
```

Comment:
In sections with multiple annotation references, I kept the ordered list to avoid rendering it as a single paragraph. If there is a single reference, I changed it to `1:`.

Reply:
Instead of this, please separate out the separate references into their own paragraphs, as [in the existing spec](https://github.com/sass/sass/blob/1c9ec00bbb061fa1be92c2e8c81144e6b8d8159b/spec/functions.md?plain=1#L66-L73)

Reply:
Updated in https://github.com/sass/sass/pull/3969/commits/63f009e0bd754eb37423a240e7b97dab6753d011

Reply:
Updated in https://github.com/sass/sass/pull/3969/commits/63f009e0bd754eb37423a240e7b97dab6753d011